spacy_llm/models/rest/anthropic/model.py

```diff
@@ -24,6 +25,11 @@ class SystemPrompt(str, Enum):
 
 
 class Anthropic(REST):
+    MODEL_NAMES = {
+        "claude-1": Literal["claude-1", "claude-1-100k"],
+        "claude-2": Literal["claude-2", "claude-2-100k"],
+    }
+
```

Comment:
We don't use `MODEL_NAMES` anymore (we have `get_model_names()` instead), this was removed on purpose.

```suggestion
```